src/main/conversions.c

```diff
@@ -144,8 +144,8 @@ as_status char_double_ptr_to_py_list(as_error *err, int num_elements,
     return err->code;
 }
 
-as_status strArray_to_py_list(as_error *err, int num_elements, int element_size,
-                              char str_array_ptr[][element_size],
+as_status strArray_to_py_list(as_error *err, int num_elements,
+                              char str_array_ptr[][AS_ROLE_SIZE],
```

Comment:
If this assumes it is working with roles I think it should be renamed so that it doesn't look general purpose.

Reply:
Got it. If I remember correctly, I made this change because the Visual Studio compiler was complaining about taking in a variable size array.